packages/marketplace-ui/src/components/MarketplaceProviders.tsx

```diff
@@ -9,7 +9,7 @@ export const MarketplaceProviders: FC<{
 }> = ({ children, onError = defaultOnError, resetCSS = false }) => (
   <StrataProviders onError={onError} resetCSS={resetCSS}>
     <MarketplaceSdkProvider>
-      <HolaplexGraphqlProvider>{children}</HolaplexGraphqlProvider>
+      <GraphqlProvider>{children}</GraphqlProvider>
```

Comment:
Major breaking change to rename this. I'd rather keep the old one around and add the new one.